Client/ClientRoom.cpp

```diff
@@ -137,12 +138,10 @@ void ClientRoom::_disconectionProcess()
     _communicatorInstance.get()->sendDataToAClient(_serverEndpoint, nullptr, 0, 13);
 }
 
-void ClientRoom::_holdGameOver()
+void ClientRoom::_holdADisconnectionRequest(CommunicatorMessage disconnectionDemand)
 {
-    std::vector<std::shared_ptr<ecs::Entity>> player = _worldInstance->joinEntities<Controlable>();
-
-    if (player.empty())
-        this->_state = ClientState::ENDED;
+    (void)disconnectionDemand;
```

Comment:
Why do you want the CommunicatorMessage if you don't use it ?

Reply:
Very good question :thinking: